libraries/AP_Scripting/examples/require-home-set-to-arm.lua

```diff
@@ -0,0 +1,20 @@
+-- example script for ensuring home is set before allowing arming
+-- useful so users don't take off in stabilize before RTL will work.
+
+---@dxiagnostic disable: param-type-mismatch
```

Comment:
you really should not need this on such a small script.

Reply:
infact, I think there is a typo anyway, so its not working.

Reply:
> infact, I think there is a typo anyway, so its not working.

I introduced the typo to see if the line was needed :-)